azerothcore/azerothcore-wotlk
# Feature/devmessage

**Changes proposed:**

Adds a message that only displays to users with admin privileges level 2 or higher (as to not nag someone that was just given level one for gm privileges.  Message reminds them to join us on discord and contribute to the project

**Tests performed:**

Compiles, installs and performs as expected on linux.  There are no files loaded or outside resources queried so this should be multifunctional across linux/mac/windows with no modifications needed.

**How to test the changes:**

Make sure you have an account that has gm level set to 2-4 and login.  You should get a yellow onscreen message reminding you about Discord and asking you to contribute to the project

**Known issues and TODO list:**

N/A
